src/Cli/dotnet/commands/dotnet-workload/install/IInstaller.cs

```diff
@@ -20,7 +20,13 @@ internal interface IInstaller : IWorkloadManifestInstaller
 
         void RepairWorkloads(IEnumerable<WorkloadId> workloadIds, SdkFeatureBand sdkFeatureBand, DirectoryPath? offlineCache = null);
 
-        void GarbageCollectInstalledWorkloadPacks(DirectoryPath? offlineCache = null);
+        void GarbageCollectInstalledWorkloadPacks(DirectoryPath? offlineCache = null, bool cleanAllPacks = false);
+
+        /// <summary>
+        /// Remove all non-VS workload installation records for the specified feature band, and all records for feature bands lower than featureBand.
+        /// </summary>
+        /// <param name="currentFeatureBand">The maximum feature band to remove all workload installation records from.</param>
+        void EradicateAllWorkloadInstallationRecords(SdkFeatureBand currentFeatureBand);
```

Comment:
It looks like this method is only called from inside each `IInstaller` implementation.  So I think it doesn't need to be added to the interface, and doesn't need to be public either.